api/newsletters/template.py

```diff
@@ -0,0 +1,401 @@
+BTN_PRIMARY = "#3498db"
+BTN_SECONDARY = "#43ff8c"
+BTN_TERTIARY = "#db344d"
+
```

Comment:
the colors used for the buttons. These variables aren't used. But listed here if a search is required